tests/test_cuaev.py

```diff
@@ -52,6 +52,36 @@ def setUp(self):
         num_species = 4
         self.aev_computer = torchani.AEVComputer(Rcr, Rca, EtaR, ShfR, EtaA, Zeta, ShfA, ShfZ, num_species)
         self.cuaev_computer = torchani.AEVComputer(Rcr, Rca, EtaR, ShfR, EtaA, Zeta, ShfA, ShfZ, num_species, use_cuda_extension=True)
+        self.nn = torch.nn.Sequential(torch.nn.Linear(384, 1, False)).to(self.device)
+        self.radial_length = self.aev_computer.radial_length
+
+    def _double_backward(self, aev_computer, species, coordinates):
+        torch.manual_seed(12345)
+        # graph1 input -> aev
+        coordinates = coordinates.clone().detach().requires_grad_()
+        _, aev = aev_computer((species, coordinates))
+        # graph2 aev -> E
+        aev_ = aev.clone().detach().requires_grad_()
+        E = self.nn(aev_).sum()
+        # graph2 backward
+        aev_grad = torch.autograd.grad(E, aev_, create_graph=True, retain_graph=True)[0]
+        # graph1 backward
+        aev_grad_ = aev_grad.clone().detach().requires_grad_()
+        force_ref = torch.autograd.grad(aev, coordinates, aev_grad_, create_graph=True, retain_graph=True)[0]
```

Comment:
Rename it to `force`, since this is a separate function, and it is not reference